hyperon_das/engines.py

```diff
@@ -0,0 +1,282 @@
+import json
```

Comment:
`engines.py` is not a good name. It's misleading as it gives the impression that any "engines" would be defined in this file, which is not true. `query_engines.py` is a better name IMO.

Reply:
@andre-senna, but what about our new engine, TraserveEngine? Don't you think this class and its concrete implementations couldn't stay in this file?

Reply:
fixed